spk/octoprint/Makefile

```diff
@@ -1,40 +1,38 @@
 SPK_NAME   = octoprint
-SPK_VERS   = $(shell date +%Y%m%d)
-SPK_REV    = 4
+SPK_VERS   = 1.4.0
```

Comment:
This will prevent automated update in Package Center.
But I prefer that way. Users still running old version installed on DSM 5 before upgrade to DSM 6 will have to download and install manually

Reply:
> This will prevent automated update in Package Center.
> But I prefer that way. Users still running old version installed on DSM 5 before upgrade to DSM 6 will have to download and install manually

I have now deactivated all existing octopus versions in the sc repository. Hopefully those will disapear in the package center on DSM before we will publish this updated version.
Otherwise it will display an update available with version 20160321-4 when you have installed version 1.7.1-6 ....